stl/inc/format

```diff
@@ -1453,32 +1515,40 @@ _OutputIt _Write_integral(_OutputIt _Out, const _Integral _Value, _Basic_format_
 // clang-format off
 template <class _CharT, class _OutputIt, integral _Integral>
     requires (!_CharT_or_bool<_Integral, _CharT>)
-_OutputIt _Write(_OutputIt _Out, const _Integral _Value, const _Basic_format_specs<_CharT>& _Specs) {
+_OutputIt _Write(_OutputIt _Out, const _Integral _Value, const _Basic_format_specs<_CharT>& _Specs, locale _Locale) {
     // clang-format on
-    return _Write_integral(_Out, _Value, _Specs);
+    return _Write_integral(_Out, _Value, _Specs, _Locale);
 }
 
 template <class _CharT, class _OutputIt>
-_OutputIt _Write(_OutputIt _Out, const bool _Value, const _Basic_format_specs<_CharT>& _Specs) {
+_OutputIt _Write(_OutputIt _Out, const bool _Value, _Basic_format_specs<_CharT> _Specs, locale _Locale) {

```

Comment:
How do we determine when to take `_Basic_format_specs` by value vs. `const` reference? I can find no pattern.

Reply:
it's by value when the function wants to modify some of them and pass the modified ones on.